src/plugin.tsx

```diff
@@ -53,6 +53,35 @@ export const pluginStandalone: BackstagePlugin = createPlugin({
 });
 
 export function getEntityMock(): Entity {
```

Comment:
I'm not sure it makes sense for these to be in this file. These would get imported into the built package this way I think.

There's an approach [msw recommend ](https://mswjs.io/docs/getting-started/mocks#mock-definition). What do you think? basically we could create a src/mocks directory and put msw stuff there. If we find we're replicating across different plugins it'd be easier to find code to pull out into our own test utils module (or add to the OSS one).